tests/providers/amazon/aws/hooks/test_batch_waiters.py

```diff
@@ -241,7 +241,7 @@ def describe_jobs_response(job_id: str = "mock-job-id", status: str = INTERMEDIA
 
         return {"jobs": [{"jobId": job_id, "status": status}]}
 
-    @pytest.mark.parametrize("status", ALL_STATES)
+    @pytest.mark.parametrize("status", sorted(ALL_STATES))
```

Comment:
You cannot use sets in parametrize parameter when runnung pytest-xdist because they are not hashable (even though Python 3.6 + sets are ordered. This break pytest-xdist execution. conveting them to sorted list helps - because sorted list has stable sequence.